orc8r/protos/mconfig/mconfigs.proto

```diff
@@ -92,6 +92,9 @@ message MagmaD {
 
     // List of dynamic_services
     repeated string dynamic_services = 10;
+
+    //orc8r version
+    string orc8r_version = 11;
```

Comment:
Can you include info here in the comments about what are the valid range of values?

Reply:
Since there's special logic for returning version `0.0.0` when we can't determine the Orc8r version, that info should also be included in this proto definition.